ros_gz_sim/ros_gz_sim/actions/gzserver.py

```diff
@@ -26,23 +26,23 @@
 from launch.substitutions import PathJoinSubstitution
 from launch_ros.substitutions import FindPackageShare
 
-@expose_action('gzserver')
+@expose_action('gz_server')
 class GzServer(Action):
-	"""Action that executes a gzserver ROS [composable] node."""
+	"""Action that executes a gz_server ROS [composable] node."""
 
 	def __init__(
 		self,
 		*,
-		world_sdf_file: SomeSubstitutionsType,
-		world_sdf_string: SomeSubstitutionsType,
-		container_name: SomeSubstitutionsType,
-		use_composition: SomeSubstitutionsType,
+		world_sdf_file: Optional[SomeSubstitutionsType],
+		world_sdf_string: Optional[SomeSubstitutionsType],
+		container_name: Optional[SomeSubstitutionsType],
+		use_composition: Optional[SomeSubstitutionsType],
```

Comment:
```suggestion
		world_sdf_file: Optional[SomeSubstitutionsType] = None,
		world_sdf_string: Optional[SomeSubstitutionsType] = None,
		container_name: Optional[SomeSubstitutionsType] = None,
		use_composition: Optional[SomeSubstitutionsType] = None,
```

Reply:
Thanks! Added in [2416aaf](https://github.com/gazebosim/ros_gz/pull/528/commits/2416aafce0bd0b4e3ff2f1ca3ecfdb3e4d2813cf).